inference-engine/ie_bridges/python/install/wheel/CMakeLists.txt

```diff
@@ -0,0 +1,104 @@
+# Copyright (C) 2018-2020 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+set(WHEEL_PACKAGE_NAME "openvino" CACHE STRING "Name of the package")
+set(WHEEL_LICENCE_TYPE "Proprietary - Intel" CACHE STRING "License for the package")
+set(WHEEL_AUTOR "Intel Corporation" CACHE STRING "Package author’s name")
+set(WHEEL_AUTOR_EMAIL "openvino_pushbot@intel.com" CACHE STRING "Email address of the package author")
```

Comment:
Is it correct email address?

Reply:
Yes, this mail is correct(we are publishing the packages using this mail), but those properties can/should be changed in our CI. So we can define any other default values.